src/components/NotePopup/NotePopup.js

```diff
@@ -78,7 +74,7 @@ function NotePopup(props) {
         className={notePopupButtonClass}
         onClick={togglePopup}
         onKeyDown={e => {
-          if (e.key === 'Enter' || e.key === ' ') {
```

Comment:
Should "space" be removed? It's usually used to activate buttons as well.

Reply:
> I can't see any prevent defaults removed or added? Furthermore, the on focus outside is nice to close the popup when you click outside or tab past it. Is this a good fix to remove that?

I changed it with the last commit to be as in their new version https://github.com/PDFTron/webviewer-ui/blob/8.10/src/components/NotePopup/NotePopup.js. 
What I had before was, I don't know if it is better or not, but they decided to remove the `useFocusOut()` hook, that's why I changed the approach and did as they did. They might have fixed the focusout in another place that I don't know of though!
<img width="548" alt="image" src="https://user-images.githubusercontent.com/61479393/202671866-632be0bb-5af6-4327-86c9-633ff1b20fd8.png">